applications/vanilla/views/discussion/helper_functions.php

```diff
@@ -615,6 +615,7 @@ function writeEmbedCommentForm() {
 
                 // If we aren't ajaxing this call then we need to target the url of the parent frame.
                 $returnUrl = $controller->data('ForeignSource.vanilla_url', Gdn::request()->pathAndQuery());
+                $returnUrl = trim($returnUrl,'/').'#vanilla-comments';
```

Comment:
I don't think our linter is enforcing this at the moment but we have a bit more white space in our expressions now.
```suggestion
                $returnUrl = trim($returnUrl, '/') . '#vanilla-comments';
```